src/core/src/lib/components/formly.field.ts

```diff
@@ -137,7 +137,11 @@ export class FormlyField implements OnInit, OnChanges, DoCheck, AfterContentInit
   }
 
   private attachComponentRef<T extends FieldType>(ref: ComponentRef<T>, field: FormlyFieldConfigCache) {
```

Comment:
Just a question never seen the format <T extends FieldType>(ref: ComponentRef<T>, field: FormlyFieldConfigCache)
Just wondering why you could not just 
private attachComponentRef <FieldType> (ref: ComponentRef<FieldType>, field: FormlyFieldConfigCache)

Reply:
just to ensure that the passed ref should extend `FieldType`, in our case 
we pass `FieldWrapper` to attachComponentRef which extend FieldType.